easybuild/easyconfigs/r/R-synchronicity-1.1.9.1-foss-2015b-R.3.2.3.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+
+easyblock = 'RPackage'
+
+name = 'synchronicity'
```

Comment:
@verdurin: sorry: should be `r/R/synchronicity...eb` rather than `r/R-synchronicity...eb`

(`r/R/` vs `r/R-`)